README.md

```diff
@@ -1,444 +1,89 @@
-
-## Getting Started
-
-We have a [Quickstart notebook][quickstart] that can be used to quickly create and train your first Multi-Agent System. For more information on how to use Mava, please view our [usage section](#usage). Finally, for more on Mava's implementation details, please visit our [documentation].
-
-## Supported Environments
-
-A given multi-agent system interacts with its environment via an `EnvironmentLoop`. This loop takes as input a `system` instance and a multi-agent `environment`
-instance which implements the [DeepMind Environment API][dm_env]. Mava currently supports multi-agent environment loops and environment wrappers for the following environments and environment suites:
-
-* [PettingZoo][pettingzoo]
-* [SMAC][smac]
-* [Flatland][flatland]
-* [2D RoboCup][robocup]
-* [OpenSpiel][openspiel]
-* [Melting pot][meltingpot]
-
-For details on how to add your own environment, see [here](https://github.com/instadeepai/Mava/blob/develop/mava/wrappers/README.md).
-
-|<img  src="docs/images/multiw_animation.gif" width="300px"/> | <img src="docs/images/sc2_animation.gif" width="200px"/>  | <img src="docs/images/flatland.gif" width="200px" />  |
-|:---:|:---:|:---:|
-|MAD4PG on PettingZoo's Multi-Walker environment. | VDN on the SMAC 3m map.| MADQN on Flatland. |
-
-|<img  src="docs/images/robocup_animation.gif" width="300px"/> |<img  src="docs/images/madqn_meltingpot_cleanup_scenario.gif" width="300px"/> |
-|:---:|:---:|
-|MAD4PG on the 2D RoboCup environment using 6 executors.| MADQN on a melting pot clean up scenario |
-
-## Tensorflow System Implementations
-
-| **Name**         | **Recurrent**      | **Continuous** | **Discrete**  | **Centralised training**  | **Multi Processing**   |
-| ------------------- | ------------------ | ------------------ | ------------------ | ------------------- | ------------------- |
-| MADQN   | :heavy_check_mark: | :x: | :heavy_check_mark: | :x: | :heavy_check_mark: |
-| MADDPG  | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark:       | :heavy_check_mark:  | :heavy_check_mark: |
-| MAD4PG   | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark:  | :heavy_check_mark: |
-| MAPPO   | :x: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| VDN   | :heavy_check_mark: | :x: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| QMIX   | :heavy_check_mark: | :x: | :heavy_check_mark:                | :heavy_check_mark: | :heavy_check_mark: |
+## Welcome to the Mava! 🦁
 
-## Jax System Implementations
+[**Installation**](#installation-)
```

Comment:
The quickstart and installation links are not doing anything on my side i.e. not moving to those points in the README

Reply:
Thanks